routers/api/actions/runner/runner.go

```diff
@@ -69,10 +69,9 @@ func (s *Service) Register(
 	labels := req.Msg.Labels
 
 	// create new runner
-	name, _ := util.SplitStringAtByteN(req.Msg.Name, 255)
 	runner := &actions_model.ActionRunner{
 		UUID:        gouuid.New().String(),
-		Name:        name,
+		Name:        base.EllipsisStringWholeWord(req.Msg.Name, 255),
```

Comment:
Why are we changing an API response in the first place?
Let's leave it to the caller to handle that case.